db/views/hmis_households_v05.sql

```diff
@@ -0,0 +1,75 @@
+-- To update, run rails generate scenic:view hmis_households --replace
+WITH tmp1 AS (
+  (
+    SELECT
+      "Enrollment"."HouseholdID",
+      "Project"."ProjectID",
+      FALSE AS wip,
+      "Project"."data_source_id",
+      "Enrollment"."EntryDate",
+      "Exit"."ExitDate",
+      "Enrollment"."DateUpdated",
+      "Enrollment"."DateCreated",
+      "Enrollment"."DateDeleted"
+    FROM
+      "Enrollment"
+      LEFT OUTER JOIN "Exit" ON "Exit"."EnrollmentID" = "Enrollment"."EnrollmentID"
+      AND "Exit"."data_source_id" = "Enrollment"."data_source_id"
+      AND "Exit"."DateDeleted" IS NULL
+      JOIN "Project" ON "Project"."DateDeleted" IS NULL
+      AND "Project"."data_source_id" = "Enrollment"."data_source_id"
+      AND "Project"."ProjectID" = "Enrollment"."ProjectID"
```

Comment:
This should check that `Enrollment.DateDeleted IS NULL` and `Project.DateDeleted IS NULL`

Reply:
Thanks! Made this change in https://github.com/greenriver/hmis-warehouse/pull/3602